### DISCUSSION TITLE
Localization / Language files

### DISCUSSION THREAD

**Discussion by BradyThe** [XurxoMF/vs-launcher#11] [General]
Very nice mod. Looks very promising.

Here is the German localization: https://gitlab.com/Brady_The/vintage-story-localization/-/blob/main/VS%20Launcher/de.json
Don't mind the added "//info" string. That's an internal comment, for any translator coming after me, to get them up to speed about the state of the translations and expected work.

Cheers.

**Comment by XurxoMF** [comment]
Hiii, thx a lot for that.
I'll add the new localization on the next update ^^
I'll also create a new wiki or chat here to post new keys so all of you that want to translate can easily access them ^^
